documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -0,0 +1,253 @@
+// Module included in the following assemblies:
+//
+// deploying/deploying.adoc
+
+[id='proc-deploy-migrate-kraft-{context}']
+= Migrating to KRaft mode
+
+[role="_abstract"]
+If you are using ZooKeeper for cluster management of your Kafka cluster, you can migrate to using Kafka in KRaft mode. 
+KRaft mode replaces ZooKeeper for distributed coordination, offering enhanced reliability, scalability, and throughput.
+
+During the migration, you install a quorum of controller nodes as a node pool, which replaces ZooKeeper for management of your cluster. 
+You enable KRaft migration in the cluster configuration by applying the `strimzi.io/kraft: migration` annotation.  
+After the migration is complete, you switch the brokers to using KRaft and the controllers out of migration mode using the `strimzi.io/kraft: enabled` annotation.
+
+Before starting the migration, verify that your environment can support Kafka in KRaft mode, as there are a number of xref:ref-operator-use-kraft-feature-gate-str[limitations].
```

Comment:
Might be worth noting that:
* Migration can be done only to dedicated controller nodes and not to mixed nodes
* During the migration, users need to run for some time the ZooKeeper and Controller nodes in parallel and should plan to have sufficient compute resources available in their cluster.